packages/@sfcc-bff/productapi/src/api/helpers/utils.js

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
This code/file already existed to integration branch.  I'm just wonder why we have a copy of the code here?

Reply:
Yes, based on a code review comment, I was recommended to use the util like in the integration to getclientConfig instead of doing it in the component itself. This is for demo to customer purpose, it will not be merge into integration, that's why some of the code is same as integration.